recipe/meta.yaml

```diff
@@ -20,26 +20,28 @@ build:
 requirements:
   build:
     - {{ compiler('c') }}
```

Comment:
Should we use `CFLAGS=-O2 `in `build.sh`, see  https://www.gaia-gis.it/fossil/freexl/file?ci=a5b4cf7b2d88ec00&name=INSTALL&ln=62

Reply:
CFLAGS should be automatically set by our compiler activation scripts. I don't see a need to override them.